.github/workflows/R-CMD-check-docker.yaml

```diff
@@ -14,6 +14,9 @@ jobs:
     container: ghcr.io/bisaloo/libbi-docker:main
     name: docker
     env:
+      # Avoid interaction requests from apt. It's better to set it here than in
+      # the Docker img: https://github.com/moby/moby/issues/4032
+      DEBIAN_FRONTEND: noninteractive
```

Comment:
Might be a problem that it's not passed to the `sudo` env in https://github.com/sbfnk/rbi/actions/runs/6081518618/job/16497330229#step:5:1481?

A (not hugely satisfactory) solution might be to install `tzdata` in the docker image instead.